css/custom.css

```diff
@@ -1,7 +1,33 @@
-.foundry {
+.foundry
```

Comment:
I think the <tr class="foundry"> designation doesn't exist anymore. Can this selector be removed?

Reply:
I dont know where else this may be used, so lets do this as part of another round.